composer.json

```diff
@@ -7,7 +7,7 @@
     },
     "require-dev": {
         "php": "^5.5|^7.0|^8.0",
-        "composer/composer": "^2.0.7",
+        "composer/composer": "^2.6.4",
```

Comment:
How about using `^2.6.5` and remove the `if` condition down here?

Reply:
Sure, if that's preferred, then we can do that as soon as we've verified that all of our environments and containers have been updated